app/grandchallenge/evaluation/templates/evaluation/combinedleaderboard_form.html

```diff
@@ -5,7 +5,7 @@
 {% load crispy_forms_tags %}
 
 {% block title %}
-    {{ object|yesno:"Update,Create" }} -{% if object %} {{ object.title|title }} -{% endif %} {{ block.super }}
+    {{ object|yesno:"Update,Create Combined Leaderboard" }} {% if object %} - {{ object.title|title }} {% else %} - Combined Leaderboards {% endif %} - {{ block.super }}
```

Comment:
Same here. This should "Leaderboards" and that also needs to be fixed in the breadcrumbs, I think.

Reply:
Done for the three templates for combinedleaderboard